lib/usd/ui/layerEditor/abstractCommandHook.h

```diff
@@ -80,7 +80,7 @@ class AbstractCommandHook
     virtual void muteSubLayer(UsdLayer usdLayer, bool muteIt) = 0;
 
     // lock or unlock the given layer
-    virtual void lockSubLayer(UsdLayer usdLayer, bool lockIt) = 0;
+    virtual void lockSubLayer(UsdLayer usdLayer, bool lockIt, bool systemLock) = 0;
```

Comment:
Could pass 0, 1, 2 like the MEL command and even reuse the suggested enum to be used instead of 0, 1, 2.